module/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,10 @@
+// Card
+
+.card{
+  padding: $card-spacer-y $card-spacer-x;
+  min-height: $card-min-height;
+}
+.card-raised{
+  background: $card-bg;
+  box-shadow: $btn-box-shadow;
```

Comment:
Shall we create new variable for $card-box-shadow and use $btn-box-shadow there ?

Reply:
Done